pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -513,7 +513,7 @@ def test_convert_json_field_to_pandas_type(self, inp, exp):
     def test_convert_json_field_to_pandas_type_raises(self, inp):
         field = {"type": inp}
         with pytest.raises(
-            ValueError, match=("Unsupported or invalid field type: {}".format(inp))
+            ValueError, match=(f"Unsupported or invalid field type: {inp}")
```

Comment:
the parens around the string here are unnecessary

Reply:
Done.